.chloggen/gh-tracing-breaking.yaml

```
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: githubreceiver
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: The `required_header` configuration option for the trace receiver has been changed to `required_headers`.
```

```suggestion
note: The `required_header` configuration option for the trace receiver has been renamed to `required_headers`.
```
Tiny nit: Just want to make it clear to users there's no functional change, just a different name.

---

it is a functional change because before you could only set 1 key-val pair, now it's multiples